docs/website/src/site/_includes/search.njk

```diff
@@ -1,4 +1,4 @@
-<input id="searchField" class="ds-sidebar-nav__search" type="search" placeholder="Search" />
+<input id="searchField" class="ds-sidebar-nav__search" type="text" placeholder="Search" aria-label="Search the documentation website content"/>
```

Comment:
why did you change the type from `search` to `text`?

Reply:
due to Michael's suggestion :) https://sanoma.slack.com/archives/C03SA9HUUA3/p1674213916908739